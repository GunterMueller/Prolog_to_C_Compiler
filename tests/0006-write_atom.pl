main :- display(65), nl,
	display([]), nl,
	display(abc), nl,
	display(X), nl.
