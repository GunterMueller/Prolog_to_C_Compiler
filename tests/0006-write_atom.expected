65
[]
abc
_0
